pkgs/development/tools/misc/lsof/default.nix

```diff
@@ -31,7 +31,7 @@ stdenv.mkDerivation rec {
 
   # Stop build scripts from searching global include paths
   LSOF_INCLUDE = "${stdenv.cc.libc}/include";
-  configurePhase = "./Configure -n ${dialect}";
+  configurePhase = "LINUX_CONF_CC=${buildPackages.stdenv.cc}/bin/cc LSOF_CC=$CC LSOF_AR=\"$AR cr\" LSOF_RANLIB=$RANLIB ./Configure -n ${dialect}";
```

Comment:
Can use `$CC_FOR_BUILD` for the first one. Probably easier to read (if not identical semantics) with exports and a multi-line string?

Reply:
d'oh, yes, much better.